packages/backend/server/src/models/user.ts

```diff
@@ -75,6 +82,29 @@ export class UserModel extends BaseModel {
     });
   }
 
+  /**
+   * Fill the public user information to the given datas with the given user field name
+   *
+   * @example
+   * ```ts
+   * const docsWithUser = await this.models.user.fillPublicUsers(docs, 'createdBy', 'createdByUser');
+   * ```
+   */
+  async fillPublicUsers<T, F extends string>(
```

Comment:
it's not the thing a model should care.
it's more like a business logic

Reply:
sure, should move it to packages/backend/server/src/core/workspaces/resolvers/doc.ts